act-rules-format.bs

```diff
@@ -52,66 +48,105 @@ The separation between atomic rules and composite rules creates a division of re
+
 
 Rule Identifier {#rule-identifier}
 ==================================
 
-An ACT Rule MUST have a unique identifier that can be any unique text value, such as plain text, URL or a database identifier.
+An ACT Rule MUST have a unique identifier that can be any unique text, such as plain text, URL or a database identifier.
 
+<div class=example>
+  <p>ACT Rules in the Auto-WCAG community groups use file names as identifiers. They include a number for a WCAG 2.1 success criterion, and a handle that includes an element name or attribute:</p>
+  <ul>
+    <li>SC1-2-3-video-alternative</li>
+    <li>SC2-2-1+SC3-2-5-meta-refresh</li>
+    <li>SC4-1-1-unique-id</li>
+  </ul>
+</div>
 
-Rule Description {#structure-description}
+
+Rule Description {#rule-description}
 =========================================
 
-An ACT Rule MUST have a description that is in plain language and provides a brief explanation of what the rule does.
+An ACT Rule MUST have a description that is in plain language, and provides a brief explanation of what the rule does.
+
+<div class=example>
+  This rule checks that the HTML page has a title
+</div>
+
+<div class=example>
+  The rule checks that for the `html` element, there is no mismatch between the primary language in non-empty `lang` and `xml:lang` attributes, if both are used.
+</div>
 
+Accessibility Requirements {#accessibility-requirements}
+========================================================
 
-Accessibility Requirements {#structure-accessibility-requirements}
-==================================================================
+Accessibility requirements are just that: A requirement that a particular web page conforms to for it to be considered accessible. A common example of accessibility requirements are the WCAG 2.1 success criteria. Some organizations have additional requirements that come from different sources, such as laws, internal standards, etc. These too are considered accessibility requirements which can be tested using the ACT Rules Format.
 
-<div class=note>
-  **Editor's note:** The ACT Task Force is looking for feedback about the use of the term "pass" in relation to rules. While rules can "pass", their corresponding accessibility requirement can fail. A section has been added to make this explicit in the rule, but we would like to know if this is sufficient.
+An ACT Rule SHOULD list an identifier and reference for each accessibility requirement that it maps to. Exactly how conformance to accessibility requirements is determined is described in the [Evaluation Instructions](#evaluation-instructions).
```

Comment:
- According to the definition of composite rules in section 3, they actually *MUST* identify the accessibility requirements they relate to (see corresponding comment above)
- There is no "Evaluation Instructions" section (anymore). I don't think that sentence is needed anyway.

Reply:
Need to remove "evaluation instructions".